addons/a11y/src/components/ColorBlindness.tsx

```diff
@@ -63,7 +63,8 @@ export class ColorBlindness extends Component<ColorBlindnessProps, ColorBlindnes
   };
 
   onVisibilityChange = (s: boolean) => {
-    if (this.state.expanded !== s) {
+    const { expanded } = this.state;
+    if (expanded !== s) {
```

Comment:
Which rule causes this? [destructuring-assignment](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md) ?

Reply:
yeah, something react destructuring props & state

Reply:
I agree with this rule mostly, though I can be a bit annoying if you only have 1 prop or 1 state property.